android/app/google-services.json

```diff
@@ -0,0 +1,29 @@
+{
```

Comment:
Ons moet net onthou om die credentials en endpoints op parameter store in die toekoms te sit